src/Raven.Studio/wwwroot/App/views/database/documents/editDocument.html

```diff
@@ -20,7 +20,7 @@
                                 <!--<span data-bind="attr: { class: connectedDocuments.currentDocumentIsStarred()?'icon-star-filled favorite':'icon-star' }"></span>-->
                             <!--</a>-->
                             <span class="obj-name on-base-background" data-bind="text: userSpecifiedId, attr: { title: userSpecifiedId }"></span>
-                            
+                            <span id="copy-document-id-icon" title="Copy document Id" data-bind="click: _.partial(toClipboard, true)"><i class="icon-copy"></i></span>
```

Comment:
let's avoid convey information about type , and rename to: copy-document-id

Reply:
+ probably we should rename from span to a to have proper cursor.

Reply:
done